src/helix-ui/elements/_hx-modal.less

```diff
@@ -0,0 +1,30 @@
+@import "./_hx-element";
+@import 'core/hx-icon';
+
+#container {
```

Comment:
With suggested Shadow DOM markup mentioned above, all `#container` styles  can be applied directly to `<hx-modal>` in the light dom (via `src/helix-ui/styles/core/hx-modal.less`)